scripts/triggers-end-to-end-tests/triggers/index.js

```diff
@@ -16,8 +25,24 @@ const PUBSUB_SCHEDULED_TOPIC = "firebase-schedule-pubsubScheduled";
 const STORAGE_FILE_NAME = "test-file.txt";
 
 const pubsub = new PubSub();
+
+// init the Firebase Admin SDK
 admin.initializeApp();
 
+// init the Firebase JS SDK
+const app = initializeApp(
+  {
+    apiKey: "fake-api-key",
+    projectId: `${FIREBASE_PROJECT}`,
+    authDomain: `${FIREBASE_PROJECT}.firebaseapp.com`,
+    storageBucket: `${FIREBASE_PROJECT}.appspot.com`,
+    appId: "fake-app-id",
+  },
+  "TRIGGERS_END_TO_END"
+);
+const auth = getAuth(app);
+connectAuthEmulator(auth, "http://localhost:9099");
```

Comment:
```suggestion
connectAuthEmulator(auth, process.env.FIREBASE_AUTH_EMULATOR_HOST);
```

This is probably less fragile in case we change the port assignment for the auth emulator.

Reply:
Agreed, I'll fix it, thanks